### PR TITLE
Create a spider to crawl URLs from a CSV file

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: csvspider.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    If you specify a protocol, column values are interpreted as domains, and
+    the specified protocol is used for requests.
+    """
+    name = 'csv'
+
+    custom_settings = {
+        'CONCURRENT_REQUESTS': 100,  # https://docs.scrapy.org/en/latest/topics/broad-crawls.html#increase-concurrency
+        'DUPEFILTER_DEBUG': True,
+        'HTTPERROR_ALLOW_ALL': True,
+        'REDIRECT_ENABLED': False,
```
Comment (lopuhin) [inline csvspider.py:20]: Why do we want to disable it?

**Review by lopuhin — APPROVED** [review]
Looks great, thanks @Gallaecio  - left one minor question but that does not block the merge.

File: csvspider.py (hunk 1 above)
Comment (Gallaecio) [reply csvspider.py:20]: I’m not sure what my thinking was. Since it can be overriden, I see no issue in removing the setting.

**Comment by lopuhin** [comment]
Thanks @Gallaecio 👍

**Pull request merged**